### DISCUSSION TITLE
Where can I download the latest version of GPRealEstate?

### DISCUSSION THREAD

**Discussion by leviandstuff** [GriefPrevention/GriefPrevention#1079] [Q&A] [answered]
I recently asked for a feature (Probably the post right below this one) and got linked to a GitHub for GPRealEstate. This seems like it would work well but I cannot find anywhere to download the latest .jar file and the Spigot page for the addon was last updated in 2014. Sorry if it's super obvious to find the latest file, I don't use GitHub.

**Comment by Jikoo** [comment, marked as answer]
Releases are [here](https://github.com/RoboMWM/GPRealEstate/releases) - GitHub has a releases tab. That said, looks like since Robo decided to just link the CI and the CI deletes artifacts older than 3? months there's no compiled release there...
I assume you don't have the ability to compile yourself, so you can [abuse JitPack for a CI](https://jitpack.io/com/github/RoboMWM/GPRealEstate/ccb60f6733/GPRealEstate-ccb60f6733.jar) instead - only Robo can actually trigger CI rebuilds on Appveyor.

**Comment by Jikoo** [comment]
Also if you have any other related questions you can pop 'em in here, I'll still get notified.

**Comment by RoboMWM** [comment]
Yea appveyor deletes after 6 months which I prefer cuz I don't need people downloading old dev builds that have been linked. That being said I'll probably eventually setup a bukkit project page for it since it's usable.
